test/Google.Api.Gax.Tests/PollingTest.cs

```diff
@@ -88,7 +82,8 @@ public PollSource(TimeSpan timeToCompletion, int finalResult)
                 FinalResult = finalResult;
             }
 
-            private int Poll(CallSettings callSettings) => Clock.GetCurrentDateTimeUtc() < CompletionTime ? 0 : FinalResult;
+            private int Poll(CallSettings callSettings) =>
+                Clock.GetCurrentDateTimeUtc() < CompletionTime? 0 : FinalResult;
```

Comment:
Ooh, yes. Although I hadn't noticed that this was just a new line break. Will just revert the bad change.